frontendservice/src/views/specificShop.vue

```diff
@@ -0,0 +1,139 @@
+/* Copyright 2020 Google LLC Licensed under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance with the License. You
+may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. */
+
+<template>
+  <div>
+    <NavBarCustomer></NavBarCustomer>
+    <cookieinfo></cookieinfo>
+    <cookieackno></cookieackno>
+    <div class="card">
+      <div class="card-content">
+        <p class="title">{{shopinfo.shop.shopName}}</p>
+        <p v-if="flag" class="title">Number of People ahead in the queue are:</p>
+        <p
+          v-else
+          class="title"
+        >Number of people currently in queue are {{shopinfo.customersInQueue}}</p>
+        <p class="subtitle">{{shopinfo.shop.shopType}}</p>
+        <p class="subtitle">{{shopinfo.shop.address}}</p>
+        <p class="subtitle">{{shopinfo.shop.phoneNumber}}</p>
+      </div>
+      <footer class="card-footer">
+        <p class="card-footer-item">
+          <button
+            class="button is-primary is-medium"
+            v-if="flag"
+            v-on:click="getShopInfo()"
+          >Refresh token</button>
+          <button class="button is-primary is-medium" v-else v-on:click="getToken()">Take token</button>
+        </p>
+        <p class="card-footer-item">
+          <button
```

Comment:
Same for lines 36 to 40.

Reply:
Same here happening automatically when I save the code in vscode